src/OpenLoco/src/Widget.cpp

```diff
@@ -897,8 +894,9 @@ namespace OpenLoco::Ui
 
         if (activated)
         {
+            static constexpr char strCheckmark[] = "\xAC";
             drawingCtx.setCurrentFontSpriteBase(Font::medium_bold);
-            drawingCtx.drawString(*rt, window->x + left, window->y + top, colour.opaque(), _strCheckmark);
+            drawingCtx.drawString(*rt, window->x + left, window->y + top, colour.opaque(), strCheckmark);
```

Comment:
this should really be using a format args and pushing the pointer to the string in the first arg.